component/org.wso2.carbon.identity.oauth2.grant.token.exchange/src/main/java/org/wso2/carbon/identity/oauth2/grant/token/exchange/utils/TokenExchangeUtils.java

```diff
@@ -980,7 +1010,7 @@ private static Map<String, String> resolveMappedLocalClaims(JWTClaimsSet claimsS
         }
 
         if (MapUtils.isEmpty(localClaims)) {
-            throw new IdentityOAuth2Exception(OAuth2ErrorCodes.INVALID_CLIENT,
+            throw new IdentityOAuth2Exception(OAuth2ErrorCodes.INVALID_REQUEST,
```

Comment:
please check and correct this in all the places. this was there in multiple places